components/Tweet.tsx

```diff
@@ -15,21 +21,36 @@ const Tweet = ({ data }: Props) => {
     created_at,
   } = data;
 
+  const stampLink = `https://tweetstamp.org/${id_str}`;
+  const tweetLink = `https://twitter.com/${screen_name}/status/${id_str}`;
+  const userLink = `https://twitter.com/${screen_name}`;
+
   return (
-    <div className="p-4">
+    <div className="tweet p-4">
       <header className="flex items-center">
         <div className="w-12">
-          <img src={avatar} alt={name} className="rounded-full" title={name} />
+          <a href={userLink} target="_blank" rel="noopener noreferrer">
+            <img src={avatar} alt={name} className="rounded-full" title={name} />
+          </a>
         </div>
-        <a
-          href={`https://twitter.com/${screen_name}`}
-          className="ml-3"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
+        <a href={userLink} className="ml-3" target="_blank" rel="noopener noreferrer">
           <h1 className="text-lg font-semibold">{name}</h1>
           <p className="text-sm text-gray-700 leading-tight">{`@${screen_name}`}</p>
         </a>
+        <div className="flex flex-grow justify-end">
+          <Clipboard data-clipboard-text={stampLink}>
```

Comment:
Noté que React-Clipboard puede recibir una función `onSuccess` que opcionalmente podría usarse si quisieras notificar al usuario que el link ha sido copiado.

Reply:
Excelente idea, voy a agregar algún detalle para dar el feedback. :+1: